src/Symfony/Component/VarDumper/Cloner/AbstractCloner.php

```diff
@@ -269,8 +263,6 @@ public function cloneVar($var, $filter = 0)
     /**
      * Effectively clones the PHP variable.
      *
-     * @param mixed $var Any PHP variable
```

Comment:
we keep those `mixed` usually

Reply:
We do? I thought I saw them disappear in other PRs. All right, I'll revert that line.